modules/account-lib/src/coin/eth/utils.ts

```diff
@@ -389,6 +390,36 @@ export function calculateForwarderAddress(contractAddress: string, contractCount
   return addHexPrefix(forwarderAddress.toString('hex'));
 }
 
+/**
+ * Calculate the forwarder v1 address that will be generated if `creatorAddress` creates it with salt `salt`
+ * and initcode `inicode using the create2 opcode
+ * @param {string} creatorAddress The address that is sending the tx to create a new address, hex string
+ * @param {string} salt The salt to create the address with using create2, hex string
+ * @param {string} initcode The initcode that will be deployed to the address, hex string
+ * @return {string} The calculated address
+ */
+export function calculateForwarderV1Address(creatorAddress: string, salt: string, initcode: string): string {
```

Comment:
i rather we start using buffers directly for parameters. then we don't have to cause from an implementation POV whether it is hex, uft-8, base64, etc... 

but not a blocker